src/test/java/uk/gov/hmcts/reform/sscs/utility/TrimStringFieldsTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.sscs.utility;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+
+
+class TrimStringFieldsTest {
+
+    TrimStringFields underTest = new TrimStringFields();

```

Comment:
Access modifiers

Reply:
access modifiers are not needed for test classes - they are not deployed to production